recipe/conda_build_config.yaml

```diff
@@ -3,6 +3,7 @@ blas_impl:
 - mkl                      # [x86 or x86_64]
 - blis                     # [x86 or x86_64]
 - accelerate               # [osx]
+- newaccelerate            # [osx]
```

Comment:
Would it make sense to version this somehow (maybe OS or library version if specified)?

Just thinking about whether we might wind up with `newnewaccelerate` in a future update 😉

Reply:
Apple calls it `NEWLAPACK`, so if they call a newer one `NEWNEWLAPACK` then we can call it `newnewaccelerate`. :D

Reply:
Fair enough :)